src/nvim/file_search.c

```diff
@@ -1509,7 +1374,25 @@ find_file_in_path_option (
     }
   }
 
-theend:
-  return file_name;
+  return NULL;
 }
 
+char_u *search_with_suffixes(char_u *suffixes, size_t l, int find_what) {
```

Comment:
This function should be marked `static`. Can `suffixes` be `const char_u *`? Please change `l` to `len` for readability.